README.md

```diff
@@ -4,9 +4,46 @@
 
 > 🚨 Note: This SDK is only accessible to customer developers with access to the Discovery API (Enterprise accounts) or partners who have been onboarded to the Security and Compliance partner program. To learn more about the Discovery APIs, please [visit our help center](https://slack.com/help/articles/360002079527-A-guide-to-Slacks-Discovery-APIs). 🚨
 
-# Using the SDK
+# Using the SDK 
```

Comment:
```suggestion
# Using the SDK
```

Reply:
Can you add these badges here like we do for slack-sdk? https://github.com/slackapi/python-slack-sdk/blob/main/README.md#python-slack-sdk

```
[![pypi package][pypi-image]][pypi-url]
[![Python Version][python-version]][pypi-url]

[pypi-image]: https://badge.fury.io/py/slack-discovery-sdk.svg
[pypi-url]: https://pypi.org/project/slack-discovery-sdk/
[python-version]: https://img.shields.io/pypi/pyversions/slack-discovery-sdk.svg
```